oeps/processes/oep-0057-core-repositories.rst

```diff
@@ -0,0 +1,166 @@
+     - Core Repositories
+   * - Last Modified
+     - 2022-03-XX
+   * - Authors
+     - Kyle McCormick <kyle@tcril.org>
+   * - Arbiter
+     - Joe Example <joe@example.com>
+   * - Status
+     - Draft
+   * - Type
+     - Process
+   * - Created
+     - 2022-03-XX
+   * - Review Period
+     - 2022-XX-XX - 2022-XX-XX
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Abstract
+--------
+
+TODO
+
+.. note::
+
+  This OEP is a big ol' work-in-progress. Feel free to leave early feedback, but please keep in mind that it's still a rough outline. Lots of this stuff is my stream-of-consciousness. It's not representative of tCRIL's or anyone else's plans/opinions. -Kyle
+
+
+Motivation
+----------
+
+There is a desire to make the `openedx GitHub organization`_ easier to manage and understand. To do this, we must be judicious about introducing new repositories and consider removing some existing repositories. At the same time, contributors to new repositories are eager to share their work with the Open edX community and, understandably, often desire for the upstream of their repository to become reflected under the GitHub organization that bears the Open edX project's name.
+
+To best accomodate both these interests in a fair and inclusive way, this OEP aims to:
+
+* establish the purpose of the openedx GitHub organization,
+* define a transparent process for the management of the organization's set of repositories, and
+* explain how community members can prepare their repositories for inclusion in the organization.
+
+Definition of Core Repositories
+-------------------------------
+
+* We define the *Core repositories of Open edX* (or in this context, "the Core") as the entire set of public repositories hosted in the `openedx GitHub organization`_.
+
+  * Phrased another way: A repository is in the Core if and only if it is public and in the openedx GitHub organization.
+
+  * Terminology notes:
+
+    * The choice of *Core* here is inspired by the Core Contributor program, which exists in part to grant community members access to repositories in the openedx GitHub organization.
+
+    * This does not preclude other, smaller "cores" of Open edX (such as a *learning core*, *core utilities*, etc.) from existing within the Core repositories.
+
+  * **What repositories are in the Core?**
+
+    * A repository *must* be in the Core if it is directly included in `latest Open edX release`_.
+
+    * A repository *must* be in the Core if:
+
+      * it is listed as a non-optional base dependency of a repository included in the latest Open edX release *and*
+
+      * it is *not* maintained as a third-party package that is unspecific to Open edX, such as Django or Python Requests.
+
+    * In addition to the repositories that must be in the Core, a repository *may* be in the Core if:
+
+      * it is of interest to more than one entity the Open edX community (TODO: this needs better criteria); and
+
+      * it has assigned `maintainers`_ who strive to:
```

Comment:
This only applies to code repos.  We also have project repos (working group backlogs, etc).